src/main/java/com/elytradev/concrete/inventory/FluidValidators.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
This class doesn't really improve comprehension - I feel like it just puts validators in harder-to-find places. If there was some kind of logical confusion - such as if ANY_FLUIDS could ever be seen to refer to an item - it might be worth it, and even then it might be better solved by switching to a prefix nomenclature, like FLUID_ANY or ITEM_ANY.

Ideally, we'd get rid of this class completely.